needle_bot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  prismatic_position_controller:
    type: effort_controllers/JointPositionController
    joint: prismatic_joint
    pid: {p: 1.0, i: 0.01, d: 0.2}
  servo_position_controller:
    type: effort_controllers/JointPositionController
    joint: motor_joint
    pid: {p: 1.0, i: 0.0, d: 0.0}